vcpkg/users/buildsystems/cmake-integration.md

```diff
@@ -264,4 +264,14 @@ See the [`--feature-flags=`](../../commands/common-options.md#) command line opt
 
 When set to `ON`, Print every call to `find_package`. Nested calls (e.g. via `find_dependency`) are indented according to nesting depth.
 
+
+### `VCPKG_LOCK_FIND_PACKAGE_<Pkg>`
+
+When this option is set, non-nested calls to [`find_package`](<https://cmake.org/cmake/help/latest/command/find_package.html>) are either
+required (`VCPKG_LOCK_FIND_PACKAGE_<Pkg>=ON`) or disabled (`VCPKG_LOCK_FIND_PACKAGE_<Pkg>=OFF`).
+
+This variable is a tool to control direct dependencies and related features in vcpkg ports
+which use the CMake build system. It can be used with [`vcpkg_check_features`](../maintainers/functions/vcpkg_check_features.md) and avoids
```

Comment:
```suggestion
which use the CMake build system. It can be used with [`vcpkg_check_features`](../../maintainers/functions/vcpkg_check_features.md) and avoids
```